rfc6962-bis.xml

```diff
@@ -994,47 +1009,50 @@ an accepted root certificate.
-        </t>
-
-        <t>
-          Outputs:
-
-          <list>
-	    <t><list style="hanging">
-	      <t hangText="leaf_index:">
-		The 0-based index of the entry corresponding to the <spanx style="verb">hash</spanx> parameter.
-	      </t>
-	      <t hangText="audit_path:">
-		An array of base64-encoded Merkle Tree nodes proving the inclusion of the chosen certificate.
-	      </t>
-	      <t hangText="tree_size:">
-		The size of the tree, in entries, in decimal.
-	      </t>
-	      <t hangText="timestamp:">
-		The timestamp, in decimal.
-	      </t>
-	      <t hangText="sha256_root_hash:">
-		The Merkle Tree Hash of the tree, in base64.
-	      </t>
-	      <t hangText="tree_head_signature:">
-		A TreeHeadSignature for the above data.
-	      </t>
-            </list></t>
+	  <list style="hanging">
+            <t hangText="Inputs:">
+	      <list style="hanging">
+		<t hangText="hash:">
+		  A base64-encoded v1 leaf hash.
+		</t>
+		<t hangText="tree_size:">
+		  The tree_size of the tree on which to base the proof, in decimal.
+		</t>
+              </list>
+              The <spanx style="verb">hash</spanx> must be calculated as defined in <xref target="tree"/>. The <spanx style="verb">tree_size</spanx> must designate an existing v1 STH. Because of skew, the front-end may not know the requested STH. In that case, it will return the latest STH it knows, along with an inclusion proof to that STH. If the front-end knows the requested STH then only <spanx style="verb">leaf_index</spanx> and <spanx style="verb">audit_path</spanx> are returned.
+            </t>
+	    <t hangText="Outputs:">
+	      <list style="hanging">
+		<t hangText="leaf_index:">
+		  The 0-based index of the entry corresponding to the <spanx style="verb">hash</spanx> parameter.
+		</t>
+		<t hangText="audit_path:">
+		  An array of base64-encoded Merkle Tree nodes proving the inclusion of the chosen certificate.
+		</t>
+		<t hangText="tree_size:">
+		  The size of the tree, in entries, in decimal.
+		</t>
+		<t hangText="timestamp:">
+		  The timestamp, in decimal.
+		</t>
+		<t hangText="sha256_root_hash:">
+		  The Merkle Tree Hash of the tree, in base64.
+		</t>
+		<t hangText="tree_head_signature:">
+		  A TreeHeadSignature for the above data.
+		</t>
+              </list>
+	    </t>
+	    <t hangText="Errors:">
+	      <list style="hanging">
+		<t hangText="hash unknown">
+		  <spanx style="verb">hash</spanx> is not the hash of a known leaf (may be caused by skew).
+		</t>
+		<t hangText="tree_size unknown">
+		  <spanx style="verb">hash</spanx> is before the latest known STH but is not from an existing STH.
```

Comment:
There is no error - this is a normal condition and a non-error return is defined.